test/extended/cmd.sh

```diff
@@ -143,3 +144,15 @@ os::cmd::expect_success "oc process -f test/extended/fixtures/image-pull-secrets
 os::cmd::try_until_text "oc get pods/my-dc-1-hook-pre -o jsonpath='{.status.containerStatuses[0].imageID}'" "docker"
 os::cmd::expect_success "oc delete all --all"
 os::cmd::expect_success "docker rmi -f ${docker_registry}/image-ns/busybox:latest"
+
+# check to make sure that service serving cert signing works correctly
```

Comment:
I would prefer this to be a standard extended test in the conformance suite.  Does it need to do a build, or was that just convenient?  I'd prefer not to require build in order to run the test.  Can we do serving certs with nginx or similar?